coinbase_ruby/README.md

```diff
@@ -0,0 +1,24 @@
+# Coinbase Ruby
+
+This sample shows a workflow, activity, and client from [Coinbase Ruby SDK](https://github.com/coinbase/temporal-ruby)
```

Comment:
> This sample shows a workflow, activity, and client from Coinbase Ruby SDK **able to** interoperate with a workflow, activity, and client from Temporal Ruby SDK

Maybe that's just me, but that reads badly.

Reply:
I think "This sample shows [X] able to interoperate with [Y]" reads ok